crypto/secp256k1/secp256k1_nocgo.go

```diff
@@ -0,0 +1,71 @@
+// +build !cgo
+
+package secp256k1
+
+import (
+	"math/big"
+
+	secp256k1 "github.com/btcsuite/btcd/btcec"
+
+	"github.com/tendermint/tendermint/crypto"
+)
+
+// used to reject malleable signatures
+// see:
+//  - https://github.com/ethereum/go-ethereum/blob/f9401ae011ddf7f8d2d95020b7446c17f8d98dc1/crypto/signature_nocgo.go#L90-L93
+//  - https://github.com/ethereum/go-ethereum/blob/f9401ae011ddf7f8d2d95020b7446c17f8d98dc1/crypto/crypto.go#L39
+var secp256k1N, _ = new(big.Int).SetString("fffffffffffffffffffffffffffffffebaaedce6af48a03bbfd25e8cd0364141", 16)
```

Comment:
Maybe this is cleaner?

```
var (
	secp256k1N     = secp256k1.S256().CurveParams().N
	secp256k1halfN = new(big.Int).Rsh(secp256k1N, 1)
)
```

ie. https://github.com/btcsuite/btcd/blob/master/btcec/btcec.go#L919

Reply:
Thanks. I agree Especially using the existing `N` is cleaner.